docker-compose.yml

```diff
@@ -27,11 +27,25 @@ slave:
         - JAVA_HOME=/etc/alternatives/java_sdk
         - SERVICE_NAME=SPACENAME-omero
         - DOCKER_FIX='   '
+    command: -labels java -labels slave -name slave -executors 4
+
+web:
+    build: web
+    environment:
+        - JENKINS_LABEL=/etc/alternatives/java_sdk
+        - JENKINS_MODE=exclusive
+        - SERVICE_NAME=SPACENAME-omero
+    links:
+        - jenkins
+    volumes:
+        - ./web:/home/omero
+    command: -labels web -name web -executors 4
```

Comment:
I was using the label here to tie jobs to the correct location rather than the name. But as long as the newer plugin works with the rest of the stack, :+1: